pkgs/openjdk_10_0_2/default.nix

```diff
@@ -1,23 +1,36 @@
-{ stdenv, bash, coreutils, findutils, gawk, gnugrep, nix, fetchurl,
-  glibcLocales, pkgconfig, gnutar, gzip, zlib, glib, glibc, setJavaClassPath, libxslt, libxml2, libX11, libXrender, libXaw, libXext, libXt, libXtst, libXi, libXinerama, libXcursor, libXrandr, alsaLib, fontconfig, freetype }:
+{ stdenv, bash, coreutils, findutils, gawk, gnugrep, nix, fetchurl, patchelf,
```

Comment:
do we still need patchelf…?

Reply:
for the linux part, patchelf is still required and it's probably better to explicitly reference it. it probably could be made optional for the darwin part of the installation, but i was too lazy to do that (since it is not really necessary).